frontend/src/components/spatialChat/conversationsList/conversationsListView.tsx

```diff
@@ -1,10 +1,22 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
+import { ChatEntry } from '../../../classes/SpatialChat';
+import { RootState } from '../../../redux/store'
 
-export const ConversationsListView: React.FunctionComponent = () =>
-(
-    <div>
-        Conversations List view
-    </div>
-);
+export const ConversationsListView: React.FunctionComponent = () => {
+    const chatList: ChatEntry[] = useSelector((state: RootState) => state.chat.chats);
+    return (
+        <div data-testid="conversations-wrapper">
+            <ul>
+                Conversations List view
+                {
+                    chatList.map((chatEntry) =>
+                        <li key={chatEntry.generateKey()}>{chatEntry.message}</li>
+                    )
+                }
```

Comment:
NOTE: This is just adding the chatEntry to the list, unformatted. Feel free to add/change this around.